### PR TITLE
ACMS-1495: PHPUnit testcases for Headless mode enablement.

### PR DESCRIPTION
**Pull request by deepakmishra2**
**Motivation**
Fixes #ACMS-1495

**Proposed changes**
PHPunit  test coverage for headless mode enablement

### CODE REVIEW

File: modules/acquia_cms_headless/tests/src/Functional/HeadlessModeEnablement.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    $btnPanes = $assertSession->elementExists('css', '.ui-dialog-buttonpane', $acmsWelcomeModal);
+    $assertSession->buttonExists('Setup Manually', $btnPanes)->press();
+
+    // Test if Headless block is available.
+    $headlessForm = $assertSession->elementExists('css', '#acquia-cms-headless-form');
+    $headlessSummary = $assertSession->elementExists('css', '.claro-details__summary');
+    $this->assertEquals('Headless', $headlessSummary->getText());
+    $assertSession->elementExists('css', '.claro-details__summary', $headlessForm)->press();
+
+    // Test if next js gets enabled or not.
```
Comment (rajeshreeputra) [inline modules/acquia_cms_headless/tests/src/Functional/HeadlessModeEnablement.php:38]: @deepakmishra2 can we include Next.js starter kit enablement in HeadlessHybridModeTest? you can rename HeadlessHybridModeTest to HeadlessModesTest and include it.
cc @vishalkhode1 @apathak18

Comment (deepakmishra2) [reply modules/acquia_cms_headless/tests/src/Functional/HeadlessModeEnablement.php]: done with these changes

**Review by rajeshreeputra — APPROVED** [review]
Changes looks good to me

**Comment by rajeshreeputra** [comment]
> Screenshot 2023-02-20 at 6 00 34 PM
> 
> I can see one failure can you please check

done

**Review by chandan-singh7929 — APPROVED** [review]

**Pull request merged**